staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5952,6 +5952,10 @@ type PodLogOptions struct {
 	// the actual log data coming from the real kubelet).
 	// +optional
 	InsecureSkipTLSVerifyBackend bool `json:"insecureSkipTLSVerifyBackend,omitempty" protobuf:"varint,9,opt,name=insecureSkipTLSVerifyBackend"`
+
+	// wait for the container to start running before streaming logs. Defaults to false.
+	// +optional
+	Wait bool `json:"wait,omitempty" protobuf:"varint,10,opt,name=wait"`
```

Comment:
kubectl changes should not be touching API. Is this really needed?

Reply:
Thanks for the review @dims! Fixed it with the latest changes.